tests/vmi_multus_test.go

```diff
@@ -131,25 +131,26 @@ var _ = Describe("Multus", func() {
 			Do()
 		Expect(result.Error()).NotTo(HaveOccurred())
 
-		// Create identical ptp crds in two different namespaces
+		// Create identical ptp crds
```

Comment:
The Commit message title is too long. Could you make it shorter and elaborate more in the message? I understand that libvirt rejects 0xFE, but I don't see why you need to change ptp-conf-tuning CRDs. I would like to learn that you 1) Are avoiding the MAC issue 2) You did it by setting static MAC address 3) You changed CRDs because of ...